_problems/fee-bumping.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: problem
+title:  "Replacement rules and Fee Bumping"
+tags: lightning
+status: open
+maintainer: LLFourn
+---
+
+Layer-2 protocols like [Lightning] treat the blockchain as a dispute resolution and settlement layer.
+Only when their counterparty is unreachable or uncooperative does a participant have to unilaterally broadcast a transaction.
+The protocol often requires a party to confirm a transaction within a certain time so the funds are distributed fairly.
+So how can layer-2 protocol designers guarantee their users are able to confirm their transaction before the deadline?
+Since miners prioritize transactions based on their feerate, the only effective way is to make sure participants can set the fee of time sensitive transactions and ideally even increase it repeatedly after the transaction has been broadcast if conditions change.
+
+There are two approaches available in Bitcoin today.
+
+1. *Replace-by-Fee* ([RBF]): The user simply signs a new transaction that is likely to replace the old one since it pays a higher fee.
+2. *Child-Pays-For-Parent* ([CPFP]): The original transaction remains unchanged but a new transaction with a higher fee is signed which spends from the old one increasing the average feerate of the pair.
```

Comment:
A third know proposal is sponsorship : https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-September/018168.html

Reply:
Right this is in the "Proposed Solutions" section.